core/server/apps/subscribers/lib/router.js

```diff
@@ -44,9 +45,13 @@ function honeyPot(req, res, next) {
     next();
 }
 
+function validateUrl(url) {
+    return validator.isEmptyOrURL(url) ? url : '';
+}
+
 function handleSource(req, res, next) {
-    req.body.subscribed_url = req.body.location;
-    req.body.subscribed_referrer = req.body.referrer;
+    req.body.subscribed_url = validateUrl(req.body.location);
```

Comment:
ok the schema validator works, but https://github.com/TryGhost/Ghost/pull/7540/files#diff-c1c7e6bde4b46c52fa17d0ee35c41a50R87 does not react on **any errors**.

in case someone add's a duplicate email, i can understand ignoring the error.
but `Validation (isEmptyOrURL) failed for subscribed_url` error, we should not ignore or?